src/lib/customPropTypes.tsx

```diff
@@ -365,7 +377,12 @@ export const collectionShorthand = every([disallow(['children']), PropTypes.arra
  * @param {string} help A help message to display with the deprecation warning.
  * @param {function} [validator] A propType function.
  */
-export const deprecate = (help, validator) => (props, propName, componentName, ...args) => {
+export const deprecate = (help: string, validator: Function) => (
+  props,
```

Comment:
types for these args are not introduced - seems that they should be like the ones that are used for the previous expression. The same applies to the `demand` function. Just to be sure, is it intentional?

Reply:
I was just fixing the types needed for `lodash` to compile correctly, I added types where I could

Reply:
got it - at the same time I would rather suggest to use consistent approach in terms of introducing types for these tree functions mentioned: currently we have types for these exact arguments introduced for ... function, but those very same ones are absent for other functions